src/ResXManager.Model/Configuration.cs

```diff
@@ -62,7 +63,14 @@ public interface IConfiguration : INotifyPropertyChanged
 
         public string? EffectiveTranslationPrefix { get; }
 
-        public PrefixFieldType PrefixFieldType { get; }
+        [Obsolete("This property is no longer used, and should not be relied upon to determine the current state. Replaced with boolean values instead")]

```

Comment:
The actual PrefixFieldType is a flags enum, so I don't see why not to simply extend this with a third flag for the new setting.